examples/pointcloud_example.py

```diff
@@ -0,0 +1,52 @@
+import pathlib
+
+import trimesh
```

Comment:
Instead of trimexh you can directly use Magnum python bindings for MeshData.
You could have the API return a MeshData instead of vertex and index arrays also.
https://doc.magnum.graphics/python/magnum/trade/MeshData/